cmd/fluxcli/main.go

```diff
@@ -0,0 +1,316 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"math"
+	"mime"
+	"net/http"
+	"os"
+	"unicode/utf8"
+
+	"github.com/influxdata/flux"
+	_ "github.com/influxdata/flux/builtin"
+	"github.com/influxdata/flux/control"
+	"github.com/influxdata/flux/csv"
+	"github.com/influxdata/flux/lang"
+	"github.com/influxdata/platform/query"
+	"github.com/pkg/errors"
+)
+
+var (
+	q     = flag.String("q", "", "flux script")
+	httpd = flag.String("http", "", "run http server on specified port")
+)
+
+func main() {
+	flag.Parse()
```

Comment:
Can be a separate PR, but I would like to rework `fluxcli` to use `spf13/cobra`:

```
fluxcli [command]

    exec   execute a flux query
    http   start a HTTP server (for REPL or Chronograf)

```